ftfy/badness.py

```diff
@@ -129,7 +130,7 @@ def sequence_weirdness(text):
     """
     text2 = unicodedata.normalize('NFC', text)
     weirdness = len(WEIRDNESS_RE.findall(chars_to_classes(text2)))
-    punct_discount = len(ENDING_PUNCT_RE.findall(text2))
+    punct_discount = sum(len(x) for x in ENDING_PUNCT_RE.findall(text2))
```

Comment:
The comment on ENDING_PUNCT_RE says "The match ends with a + so that we only give the bonus once for a consecutive sequence of these characters."  Isn't this change just undoing that?